src/libraries/Common/src/Interop/Windows/SspiCli/Interop.SSPI.cs

```diff
@@ -315,6 +316,23 @@ public SecBufferDesc(int count)
             }
         }
 
+        [StructLayout(LayoutKind.Sequential)]
+        internal unsafe struct SecPkgCred_ClientCertPolicy
+        {
+            public int dwFlags;
+            public fixed byte guid[16];
+            public int dwCertFlags;
+            public int dwUrlRetrievalTimeout;
+            [MarshalAs(UnmanagedType.Bool)]
+            public bool fCheckRevocationFreshnessTime;
```

Comment:
```suggestion
            public BOOL fCheckRevocationFreshnessTime;
```
And also add Interop.BOOL.cs to the project file